### ISSUE TITLE
[Home page][content][images] Middle buckets

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website-design#82]
Ideas:
* Accessibility Principles - because it's *really* helpful and we've never promoted it (or maybe this should be the top one, until Easy Checks is updated)
* [Web Accessibility First Aid: Approaches for Interim Repairs](https://www.w3.org/WAI/impl/improving)
* WCAG 2.1

**Comment by shawna-slh** [comment]
spreadsheet for latest: https://docs.google.com/spreadsheets/d/1oYO8xsoKHNWkxZ3IxOv3bOjhqV3dQddjRID5XCk_27Y/edit?usp=sharing

(doc with earlier ideas: https://docs.google.com/document/d/1ofzqxTzj_VZ741-Cki2WpMMPCiZBhwNIu8JBWfTyW3g/edit?ts=59b92918# )

**Comment by shawna-slh** [comment]
Brent and Shawn worked on content for the middle boxes.

Brent called them "**Showcase Containers**" and says:
> Showcase both high priority resources or tasks -AND- most popular resources

Brent's initial ideas to get discussion started:
> **Tutorials**
A collection of tutorials on how to develop web content that is accessible to people with disabilities, and provide a better user experience for everyone.
> **Web Accessibility First Aid**
Approaches for Interim Repairs - An overview with pointers to help address the most critical issues first, and where to go from there.
> **Accessibility Principles**
An introduction to some of the web accessibility requirements for websites, web applications, browsers, and other tools, including references to the international standards.
> **How To Meet WCAG 2.0**
Also known as “QuickRef” A customizable quick reference to Web Content Accessibility Guidelines (WCAG) 2.0 requirements (success criteria)
> **Presentation You Can Use**
Accessibility presentation materials designed for self-study, as well as for advocates to support the design and development of accessible sites and applications.
> **Laws & Policies**
Governmental policies related to web accessibility. Also, a place to submit additional laws or policies not yet listed.

Shawn suggested:
> I like having the topic &/or primary audience at the top of the box, instead of the doc name. However, we'd need to make it clear that this is just *one* featured resource for that topic/audience, not all or the main thing for it/them.

Potential topics are here: <https://docs.google.com/spreadsheets/d/1oYO8xsoKHNWkxZ3IxOv3bOjhqV3dQddjRID5XCk_27Y/edit#gid=0

Second pass at content to start with:

**Getting Started** Featured Resource
If you need to urgently address accessibility in an existing web project, see [Web Accessibility First Aid: Approaches for Interim Repairs](https://www.w3.org/WAI/impl/improving).

**Basics** Featured Resource
For an introduction to requirements for websites, web applications, browsers, and other tools, see [Web Accessibility Principles](https://www.w3.org/WAI/intro/people-use-web/principles).

**Standards** Featured Resource
For a customizable checklist of Web Content Accessibility Guidelines requirements (success criteria), [see How to Meet WCAG 2.0](https://www.w3.org/WAI/WCAG20/quickref/).

**WCAG 2.1 Update**
To learn more about the status of the WCAG 2.1 Working Draft and plans for completing it in 2018, see [What about WCAG 2.1](@@)?

**Alicia<-James:** I think this approach would need some visual design tweaks from Alicia...

**Comment by yatil** [comment]
Can we _not_ use phrasing like “see [link to resource]” and instead use a contemporary method of outright linking to resources for the sake of accessibility (easier to understand and easier to click areas)?

As the items are on the homepage, it is clear that they are featured. That’s the point of the home page :-D

**Comment by shawna-slh** [comment]
@yatil, We could still have the whole area clickable.

**Comment by yatil** [comment]
@slhenry But then I think the title we’re linking to should be the first thing in the link.

**Comment by bakkenb** [comment]
+1 to @yatil

**Comment by lakeen** [comment]
+1

**Comment by VivienneConway** [comment]
+1

**Comment by shawna-slh** [comment]
@yatil I don't understand "But then I think the title we’re linking to should be the first thing in the link."

**Comment by yatil** [comment]
@slhenry Instead of having “Something Something Something, see Resource XYZ.” and linking the whole thing, we should have “ResourceXYZ Something Something Something” – basically keeping the current structure.

**Comment by James-Green** [comment]
mentioning @aliciafrausto to see if she'll get an email about this issue while we wait to get her in this group

**Comment by aliciafrausto** [comment]
I've created a mockup with the 2 ideas I see in this string. 
https://visa.invisionapp.com/share/KTDO6S6VS#/255341903_column_Alternates

I've removed the icons as I'm not sure they are relevant with the new content.

**Comment by bakkenb** [comment]
Thanks for the visualization @aliciafrausto. 

In my opinion, I do not like Option B for a few reasons.

1. I don't like "Featured Resource" repeated at the top if each box. I understand that it may be important for people to know this is a featured resource so they won't be too upset when these change from time to time and they need to get used to using the top navigation to find the resource (or just use their bookmark). If we really need to indicate that these are featured and will be changing, then I would suggest putting a title (heading x) above the four boxes that says "More Featured Resources" or something like that.
2. If "Featured Resource" needs to stay in each box, then I don't like following it with the category. When doing so, it seams like that is the name of the featured resource. Then there is a link with what seems to be another name of a resource (which of course is the actual name). Either way, I am strongly concerned that people will start calling the resource by the category instead of the proper title. Then when it is moved, they will ask us, "Where did the Basics resource go?" or "Where did the Getting Started resource go?"

In Option A, I like that it does not say "Featured Resource." Again, I think best to put that as one title above these boxes. In addition, I think it would be good to add a bit to the categories in Option A to make them more actionable. For example:
- instead of "Getting Started" maybe "Help Getting Started"
- instead of "Basics" maybe "Learn the Basics"
- instead of "Standards" maybe "Reference the Standards"
- "WCAG 2.1 Update" is fine as is

I do think it would be good to have small icons for these boxes as it draws the attention of the eye. Also they seem more like four distinct boxes for some reason when the icons are there. I agree with Alicia that the current icons don't fit/match. New icons would have to be created.

**Comment by yatil** [comment]
The content of the individual boxes is irrelevant at this stage. What should be the approach?

* Option A has the resource in large heading text and links to it.
* Option B has the category as the large heading text but links to the resource that is underlined.

We already have this: https://w3c.github.io/wai-website-components/components/detail/teaser-four-teasers.html

I can easily add a black subtitle to it, just tell me where it should go. (I think it fits nicely in-between the icon and the title.) 

This is very late in the game, at least for the UT and we need decisions asap if we want to have a chance of having them implemented.

**Comment by bakkenb** [comment]
I know I am not the only one that should weigh in on this... hoping other will too. 

But I will restate what I said in earlier comment, I prefer Option A.

**Comment by shawna-slh** [comment]
Thank you, Alicia, for mocking up ideas. It helps to see concrete examples.

**Comment by shawna-slh** [comment]
We're shoe-horning, and it's not fitting.

**Comment by yatil** [comment]
I changed the content to what was provided to me. The texts are now weirdly doubling the titles and we don’t have proper icons to go with each item, but it’s the only thing I can do for now.

Concrete changes welcome.

**Comment by yatil** [comment]
http://w3c.github.io/wai-website/

**Comment by shawna-slh** [comment]
moot now with latest direction of home page. phew :-)
